src/main/java/com/wavefront/sdk/entities/histograms/WavefrontHistogramImpl.java

```diff
@@ -257,10 +263,12 @@ public double stdDev() {
       while (binsIter.hasNext()) {
         MinuteBin minuteBin = binsIter.next();
         if (minuteBin.minuteMillis < cutoffMillis) {
-          minuteBinToCentroidsMap.putIfAbsent(minuteBin.minuteMillis, new ArrayList<>());
-          minuteBinToCentroidsMap.get(minuteBin.minuteMillis).addAll(
-                  minuteBin.distribution.centroids().stream().
-                          map(c -> new Pair<>(c.mean(), c.count())).collect(Collectors.toList()));
+          if (minuteBin.distribution.centroidCount() > ACCURACY * RECOMPRESSION_THRESHOLD_FACTOR) {
```

Comment:
Should we keep the 5 times logic align with here https://github.com/wavefrontHQ/java-lib/pull/50/files#diff-07f8313afa502162f5fdea1a3d760593R221 and system as well?

Reply:
we use `2` in java-lib as well now